src/screens/Home/index.tsx

```diff
@@ -44,19 +44,20 @@ const Home: FC = () => {
   }, [location.search]);
 
   if (weatherSelector?.WeatherState?.isFetching)
-    return <Preloader inscription='LOOKING OUTSIDE FOR YOU... ONE SEC' />;
+    return (
+      <S.Wrapper time={weatherData?.location?.localtime || ''}>
+        <Preloader inscription='LOOKING OUTSIDE FOR YOU... ONE SEC' />
+      </S.Wrapper>
+    );
 
   return (
     <S.Wrapper time={weatherData?.location?.localtime || ''} maxWidth='sm'>
       <Search errorText={weatherData?.error?.message} />
```

Comment:
{weatherSelector?.WeatherState?.isFetching && (
<S.Wrapper time={weatherData?.location?.localtime || ''}> // should be absolute/fixed
        <Preloader inscription='LOOKING OUTSIDE FOR YOU... ONE SEC' />
      </S.Wrapper>
)}